src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1896,7 +1896,7 @@ protected override void BeginProcessing()
             else
             {
                 // Working Directory not specified -> Assign Current Path.
-                startInfo.WorkingDirectory = ResolveFilePath(this.SessionState.Path.CurrentFileSystemLocation.Path);
+                startInfo.WorkingDirectory = PathUtils.ResolveFilePath(this.SessionState.Path.CurrentFileSystemLocation.Path, this, true);
```

Comment:
this should probably be done in the helper function where *no* globbing should be done. I would suggest that `ResolveFilePath` (line 2989) should probably be calling the literalpath overload.
`string filepath = PathUtils.ResolveFilePath(path, this, true)`
I don't think _any_ of those files should be globbed

Reply:
@JamesWTruher I completely agree and that was the original version of this change (I force pushed overwrote the commit history, sorry).  The the committee determined that would be a breaking change since [presumably] somebody might have gone out of their way to escape those paths in the past. The recommended approach was globbing first and then, if no match was was found, doing a literal check.  Given my lack of enthusiasm for that approach as well as the fact there was no good helper function suited to doing that (that I know of), I scaled back the change.